assets/control-plane/prometheus-rule.yaml

```diff
@@ -336,7 +336,7 @@ spec:
         description: The kubernetes apiserver has terminated {{ $value | humanizePercentage }} of its incoming requests.
         summary: The kubernetes apiserver has terminated {{ $value | humanizePercentage }} of its incoming requests.
       expr: |
-        sum(rate(apiserver_request_terminations_total{job="apiserver"}[10m]))  / (  sum(rate(apiserver_request_total{job="apiserver"}[10m])) + sum(rate(apiserver_request_terminations_total{job="apiserver"}[10m])) ) > 0.20
+        sum by(cluster) (rate(apiserver_request_terminations_total{job="apiserver"}[10m])) / ( sum by(cluster) (rate(apiserver_request_total{job="apiserver"}[10m])) + sum by(cluster) (rate(apiserver_request_terminations_total{job="apiserver"}[10m])) ) > 0.20
```

Comment:
unrelated to this PR but I'd recommend to remove the `KubeAPITerminatedRequests` alerting rule since we're not owners for the API server. And same goes with `KubeAPIDown`.

Reply:
ACK, I'll open up a new PR.